.github/workflows/test.yml

```diff
@@ -0,0 +1,13 @@
+name: Test
+
+on:
+  - pull_request
+  - push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm install
+      - run: npm test
```

Comment:
This config file is quite verbose. Is there no way we can simplify it?

Reply:
Is that comment for `test.yml` or `deployment.yml`?

Because if you _are_ referring to `npm install` and `npm test` being verbose, I think that could be compressed to `npm ci`.

Reply:
Both files and I'm talking in general, not just this line.

Reply:
Could try shaving off a couple of lines here and there, IMO if not here, we'd be moving the config somewhere else.

Pinging @fregante as well, as this is almost the same config we use in RGH.

Reply:
I think this is as short as can be. The only thing that could be moved or removed is the line to create the TXT file.

Reply:
>  The only thing that could be moved or removed is the line to create the TXT file.

Yup, could be, but also the one where we upload and download artifacts takes ~8 lines. If only we had async steps in actions.

Reply:
> If only we had async steps in actions.

Sadly not. Clear, parallel jobs are the reason why all of this code is a workflow and not in a package called from npm script.

If they allowed creating "jobs" dynamically that'd change the whole thing: one shared action could replace a whole workflow.

Reply:
Might be a good idea to wait for https://github.com/notlmn/browser-extension-template/pull/39

Reply:
@sindresorhus would like https://github.com/notlmn/browser-extension-template/pull/39